### PR TITLE
Add my module - Veslino

### PR DESCRIPTION
**Pull request by veslino-marc**
It includes:

1. Basic Information
2. Goals
3. Guessing Game

### CODE REVIEW

File: ourpackage/veslino.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+def marc():
+    os.system("cls")
+    while True:
+        print(f"\033[1;35m{'=' * 50}")
+        print("Hello, I'm Marc D. Vesliño")
+        print("1. Basic Information")
+        print("2. Goals")
+        print("3. Guessing Game")
+        print("0. Exit")
```
Comment (barcelos-kevin) [inline ourpackage/veslino.py:16]: Wait. I will add my comment. - Barcelos

**Review by barcelos-kevin — CHANGES_REQUESTED** [review]

File: ourpackage/veslino.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+def marc():
+    os.system("cls")
+    while True:
+        print(f"\033[1;35m{'=' * 50}")
+        print("Hello, I'm Marc D. Vesliño")
+        print("1. Basic Information")
+        print("2. Goals")
+        print("3. Guessing Game")
+        print("4. Comment from Barcelos")
+        print("0. Exit")
```
Comment (consultado-kirby) [inline ourpackage/veslino.py:16]: I will add my comment. - Consultado

**Review by consultado-kirby — CHANGES_REQUESTED** [review]

File: ourpackage/veslino.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    os.system("cls")
+    while True:
+        print(f"\033[1;35m{'=' * 50}")
+        print("Hello, I'm Marc D. Vesliño")
+        print("1. Basic Information")
+        print("2. Goals")
+        print("3. Guessing Game")
+        print("4. Comment from Barcelos")
+        print("5. Comment from Consultado")
+        print("0. Exit")
```
Comment (andaya-gener) [inline ourpackage/veslino.py:16]: I will add my comment. - Andaya

**Review by andaya-gener — CHANGES_REQUESTED** [review]

File: ourpackage/veslino.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    while True:
+        print(f"\033[1;35m{'=' * 50}")
+        print("Hello, I'm Marc D. Vesliño")
+        print("1. Basic Information")
+        print("2. Goals")
+        print("3. Guessing Game")
+        print("4. Comment from Barcelos")
+        print("5. Comment from Consultado")
+        print("6. Comment from Andaya")
+        print("0. Exit")
```
Comment (piadozo-edriane) [inline ourpackage/veslino.py:16]: I would like to add a comment about your program - Piadozo

**Review by piadozo-edriane — CHANGES_REQUESTED** [review]

File: ourpackage/veslino.py (hunk 2 above)
Comment (veslino-marc) [reply ourpackage/veslino.py:16]: You have done it.

File: ourpackage/veslino.py (hunk 4 above)
Comment (veslino-marc) [reply ourpackage/veslino.py:16]: You have done it.

File: ourpackage/veslino.py (hunk 1 above)
Comment (veslino-marc) [reply ourpackage/veslino.py:16]: You have done it.

**Review by andaya-gener — APPROVED** [review]

**Review by consultado-kirby — APPROVED** [review]

**Review by barcelos-kevin — APPROVED** [review]

**Review by piadozo-edriane — APPROVED** [review]
Thank you :)

**Pull request merged**